src/main.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```

Comment:
No need of importing the react package instead you can do a named import of the strict mode from the react 
```suggestion
import {StrictMode} from 'react'
```